Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:focal as builder
+LABEL maintainer "who_should_this_be@gmail.com"
+LABEL description="This image contains the cargo contract command"
+
+ARG RUSTC_VERSION="nightly-2020-10-27"
+ENV RUSTC_VERSION=$RUSTC_VERSION
+ENV PROFILE=release
+ENV PACKAGE=polkadot-runtime
+
+RUN mkdir -p /cargo-home /rustup-home
+WORKDIR /build
+ENV RUSTUP_HOME="/rustup-home"
+ENV CARGO_HOME="/cargo-home"
+ENV DEBIAN_FRONTEND="noninteractive"
+
+# We first init as much as we can in the first layers
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install --no-install-recommends -y \
+        cmake pkg-config libssl-dev \
+        git clang bsdmainutils jq ca-certificates curl binaryen && \
```

Comment:
AFAIU this version of `binaryen` for Ubuntu:focal is not enough to run `contracts`.